src/core/app/controllers/anthropic_controller.py

```
@@ -533,8 +533,39 @@ def get_anthropic_controller(service_provider: IServiceProvider) -> AnthropicCon
 
                 session_manager = SessionManager(session, session_resolver)
+                wire_capture = None
+                try:
+                    from src.core.interfaces.wire_capture_interface import (
+                        IWireCapture as _IWireCapture,
+                    )
+
+                    wire_capture = service_provider.get_service(
+                        cast(type, _IWireCapture)
+                    )
+                except Exception:
```

Ruff (0.13.3)

Do not catch blind exception: `Exception`

(BLE001)